src/main/java/org/isf/stat/reportlauncher/gui/ReportLauncher.java

```diff
@@ -234,9 +219,53 @@ private JPanel getJParameterSelectionPanel() {
 			JLabel jRptLabel = new JLabel(MessageBundle.getMessage("angal.stat.report"));
 
 			jRptComboBox = new JComboBox<>();
-			for (String[] matrix : reportMatrix) {
-				jRptComboBox.addItem(MessageBundle.getMessage(matrix[BUNDLE]));
-			}
+			jasperFilesInFolder = new LinkedList<>();
+			folderNameFileNameMap = new HashMap<>();
+			try {				
+				List<File> loadedFiles = Files.walk(Paths.get("./rpt_stat"))
+								.filter(Files::isRegularFile)
+                                .map(Path::toFile)
+                                .filter(t -> t.getName().endsWith(".jasper"))
+                                .collect(Collectors.toList());
+				jasperFilesInFolder.addAll(loadedFiles);
+				folderNameFileNameMap.put("rpt_stat", loadedFiles.stream().map(t -> t.getName().replace(".jasper", "")).collect(Collectors.toList()));
+			
+				loadedFiles = Files.walk(Paths.get("./rpt_extra"))
+                                .filter(Files::isRegularFile)
+                                .map(Path::toFile)
+                                .filter(t -> t.getName().endsWith(".jasper"))
+                                .collect(Collectors.toList());
+				jasperFilesInFolder.addAll(loadedFiles);
+				folderNameFileNameMap.put("rpt_extra", loadedFiles.stream().map(t -> t.getName().replace(".jasper", "")).collect(Collectors.toList()));
+			
+				reportNameFileMap = new HashMap<>();
+				for(File f: jasperFilesInFolder) {
+					try {
+						Properties props = ConfigurationProperties.loadPropertiesFile(
+										f.getName().replace(".jasper", "_"+(new Locale(GeneralData.LANGUAGE)).getLanguage()+".properties"), LOGGER);
+						
+						if(props != null && props.getProperty("jTitle") != null && !props.getProperty("jTitle").isEmpty()) {
```

Comment:
```suggestion
						if (props != null && props.getProperty("jTitle") != null && !props.getProperty("jTitle").isEmpty()) {
```

Reply:
It seems that the rebase command did not update the file with the changes made by David. Now I have aligned the two files